dozer-api/src/grpc/types_helper.rs

```diff
@@ -145,9 +146,9 @@ pub fn map_field_definitions(
 fn field_type_to_internal_type(typ: FieldType) -> Type {
     match typ {
         FieldType::UInt => Type::UInt,
-        FieldType::U128 => Type::UInt, // TODO: Map this correctly
+        FieldType::U128 => Type::String,
```

Comment:
Sorry just saw this. We need to add `Type::U128` and `Type::I128`.

Reply:
so we going to add U128 and I128 instead of String right?